src/styles/scss/tools/variables.fonts.scss

```diff
@@ -1,44 +1,41 @@
 $font__title: "Lora", serif;
-$font__body: "Noto Sans JP", sans-serif;
+$font__body: "Noto Sans", sans-serif;
 $font__accent: "Gochi Hand", cursive;
 $font__weight--normal: 400;
 $font__weight--semi-bold: 500;
 $font__weight--bold: 700;
 
-// Since we host the fonts ourselves, they may need to be updated in the future
-// We use this online open source tool to retrieve fonts & their css definition:
-// https://gwfh.mranftl.com/fonts
```

Comment:
Is there a reason you dropped this comment?

If you used this tool to generate the new font definitions then I think it is very appropriate to keep around.

Reply:
I did not use what was commented before to generate the new fonts. I used [font squirrel](https://www.fontsquirrel.com/) this time, but there are other generators. 

I don't think there is a particular reason to use their old generator, but that every generator is fine. As such i feel the comment doesn't provide anything, but if you want i do not mind adding a comment explaining these are generated with fontsquirrel etc. like the old comment.

Reply:
I think there is value in providing information about the font generator used since this in the end can effect how the font is rendered to browsers.

- Google Web Fonts Helper includes version information about fonts in the generator and in the file name. This makes it possible to check if we are using a different versions and potentially track changes to a changelog.
- Font Squirrel has a lot of customization options. For future reference it is important to know what options the current webfonts were generated with - basic, optimal or expert. If expert even more information may be required :smile:.

All in all font generation can be a somewhat opaque process. If we are updating fonts and seeing unexpected changes it is important to be able to limit the factors which may cause these.